### PR TITLE
Janus build change

### PR DESCRIPTION
**Pull request by andriy-andreyev**

### CODE REVIEW

File: ivshmem/test/ivshmem-test.vcxproj [hunk 1]
```diff
@@ -18,28 +18,25 @@
     <ProjectGuid>{87AEE822-FF8B-4743-A567-8B46F4FA6D99}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>ivshmemtest</RootNamespace>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
```
Comment (kostyanf14) [inline ivshmem/test/ivshmem-test.vcxproj:21]: Please add info about this patch into commit message. This is not clear why you update only one repository.

File: Janus-build.bat [hunk 2]
```diff
@@ -8,8 +8,13 @@ SET Feature_AlwaysDefaultVendor=false
 SET _VENDOR_=JAN
 ::version for Win10 1.0.N.100
 SET JAN_RELEASE_N=1
-call tools\build.bat vioserial\vioser.sln Win10 x64
-if errorlevel 1 goto :eof
+set PROJS=vioserial\vioser.sln ivshmem\ivshmem.sln
```
Comment (kostyanf14) [inline Janus-build.bat:11]: Please check that ivshmem have proper build config for `SET _VENDOR_=JAN` and set valid copyright info

File: ivshmem/test/ivshmem-test.vcxproj (hunk 1 above)
Comment (andriy-andreyev) [reply ivshmem/test/ivshmem-test.vcxproj:21]: Ok

File: Janus-build.bat (hunk 2 above)
Comment (andriy-andreyev) [reply Janus-build.bat:11]: Checked, everything is OK

**Review by ybendito — APPROVED** [review]

**Pull request merged**